shell/platform/darwin/ios/BUILD.gn

```diff
@@ -305,17 +305,26 @@ copy("copy_license") {
   outputs = [ "$root_out_dir/LICENSE" ]
 }
 
-shared_library("copy_and_verify_framework_headers") {
+shared_library("copy_and_verify_framework_module") {
+  framework_search_path = rebase_path("$root_out_dir")
   visibility = [ ":*" ]
-  include_dirs = [ "$_flutter_framework_headers_copy_dir" ]
+  cflags_objc = [
+    "-F$framework_search_path",
+    "-fmodules",
+    "-Wnon-modular-include-in-framework-module",
+  ]
 
   sources = [ "framework/Source/FlutterUmbrellaImport.m" ]
-  deps = [ ":copy_framework_headers" ]
+  deps = [
+    ":copy_framework_headers",
+    ":copy_framework_info_plist",
+    ":copy_framework_module_map",
+  ]
```

Comment:
Ah the `gn` formatter points out I need to set up the modulemap/headers/info.plist deps here, the order in `flutter_framework` doesn't matter.